packages/website/src/components/layout/web/primary/cta/SearchButton.tsx

```diff
@@ -0,0 +1,59 @@
+'use client'
+
+import Modal from '@mui/material/Modal'
+import Image from 'next/image'
+import { usePathname } from 'next/navigation'
+import { useCallback, useState } from 'react'
+
+import discordIconImg from '@/app/search.svg'
+import Search from '@/components/layout/web/primary/cta/Search'
+
+export default function SearchButton () {
+  const path = usePathname()
+  const pathSegments = path.split('/')
+
+  // Only show if on a docs page
+  const docsShowing = pathSegments.length >= 2 && pathSegments[1] === 'docs'
```

Comment:
An element should not be responsible for determining if it is visible or not. The parent should.

Reply:
I was thinking the same but saw the version dropdown component doing this and followed it. I will bring it out unless there was a specific reason for the version component.

Reply:
moved.